src/script/lua_api/l_object.cpp

```diff
@@ -2561,6 +2570,16 @@ int ObjectRef::l_get_lighting(lua_State *L)
 	lua_pushnumber(L, lighting.shadow_intensity);
 	lua_setfield(L, -2, "intensity");
 	lua_setfield(L, -2, "shadows");
+	lua_newtable(L); // "ambient_light"
+	lua_pushnumber(L, lighting.ambient_light.getRed());
+	lua_setfield(L, -2, "r");
+	lua_pushnumber(L, lighting.ambient_light.getGreen());
+	lua_setfield(L, -2, "g");
+	lua_pushnumber(L, lighting.ambient_light.getBlue());
+	lua_setfield(L, -2, "b");
+	lua_pushnumber(L, 255);
+	lua_setfield(L, -2, "a");
```

Comment:
Is the alpha field here required for something or can it be omitted?

Reply:
It is required to be set also because the color is returned in the table form of the `ColorSpec`. The table form assumes the alpha availability, otherwise it will be nil.